src/eradiate/srf_tools.py

```diff
@@ -0,0 +1,824 @@
+"""Spectral response function (SRF) filtering algorithms."""
+import datetime
+import typing as t
+import warnings
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pint
+import rich
+import xarray as xr
+from rich.prompt import Confirm
+from rich.table import Table
+
+from eradiate import __version__
+
+from . import converters
+from .typing import PathLike
+from .units import to_quantity
+
+
+def update_attrs(srf: xr.Dataset, filter_name: str, filter_attr: str) -> None:
+    """
+    Update data set attributes to indicate that a filtering operation occurred.
+
+    Parameters
+    ----------
+    srf: Dataset
+        Data set whose attributes to update.
+
+    filter_name: str
+        Filter name.
+
+    filter_attr: str
+        Content of the 'filter' attribute.
+
+    Notes
+    -----
+    Update inplace.
+    A ``filter`` attribute is added if it did not already exist. If it existed,
+    it is updated by appending a new line to it.
```

Comment:
Conciseness:
```suggestion
    * The update is done inplace.
    * If the dataset has no ``filter`` attribute, it is created. Otherwise, a new line is 
      appended to it.
```